README.md

```diff
@@ -1,8 +1,40 @@
-How to install this repo:
+InfraSIM -- Legacy hardware Emulator
+-----------------------------------------------------
 
-1. sudo apt-get install socat ipmitool ipmiutil qemu openipmi python-pip libssl-dev libssh-dev
-2. sudo pip install -r requirements.txt
-3. sudo python setup.py install
-4. sudo infrasim-init
-5. sudo infrasim-main start
+System Basic Requirements
+-------------------------
+Infrasim package can be installed in any physical machine or virtual machines hosted by Virtualbox, ESXi, Parallel Desktop or cloud provider like AWS, Linode and etc.
 
+The basic installation system requirements are:
+    1.  Ubuntu Linux 64bit OS (14.04/15.04/16.04)
+    2.  at least 2GB memory
```

Comment:
2GB memroy is for: 1GB for host system, 1GB for qemu.
So, I think it's the minimal memory requirements.